flow/project/cli/config/manipulators/composer.go

```diff
@@ -0,0 +1,210 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package manipulators
```

Comment:
No, it's not but else I can't get rid of circular dependecy

Reply:
It's just a bit confusing that `manipulators` is now main entry point for loading config.

Is the circular dependency just from the `json` package? 

If so, maybe this could work?

```
/config
  /json
  composer.go
  processor.go
  config.go
```

```go
// instantiate composer with supported parsers
composer := config.NewComposer(json.NewParser())
```

The caller has to import two packages, but I think this is a bit easier to understand

Reply:
I had it like this before and I get:
```
package github.com/onflow/flow-cli/flow/project/cli/config (test)
        imports github.com/onflow/flow-cli/flow/project/cli/config/json
        imports github.com/onflow/flow-cli/flow/project/cli/config: import cycle not allowed in test
```

Actually, nope, it's fine, that was only a problem in a specific test and I thought is a problem in general